python/sglang/srt/model_executor/model_runner.py

```diff
@@ -462,6 +467,15 @@ def init_torch_distributed(self):
         set_custom_all_reduce(not self.server_args.disable_custom_all_reduce)
 
         if not self.is_draft_worker:
+            if self.device == "cpu":
+                import sgl_kernel
+
+                # Bind OpenMP threads to CPU cores
+                assert (
+                    cpu_has_amx_support()
+                ), "init_cpu_threads_env failed since intel amx backend is not available"
+                torch.ops.sgl_kernel.init_cpu_threads_env(self.local_omp_cpuid)
```

Comment:
this line need privileged mode enabled for both docker and k8s pod. For enterprise env, it is not allowed. it will be a road blocker for enterprise deployment.